.github/workflows/audit.yml

```diff
@@ -3,11 +3,11 @@ name: NPM Audit
 on: [push]
 
 jobs:
-  audit:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v1
-      - uses: actions/setup-node@v1
-        with:
-          node-version: 12
-      - run: npm audit
+    audit:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v1
+            - uses: actions/setup-node@v1
+              with:
+                  node-version: 12
```

Comment:
**not part of this change**: we should update our github workflows to test with node 14 instead of 12

Reply:
done